### PR TITLE
[fa] Update length of password in passwords.php

### PR DESCRIPTION
**Pull request by mshahamirian**
In laravel 5.8.15 the minimum length for password is eight characters.

### CODE REVIEW

**Comment by andrey-helldar** [comment]
For Laravel 5.7 and below, use version `^3.0`.
For Laravel 5.8, use version `^4.0`.

File: src/fa/passwords.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
     |
     */
 
-    'password' => 'گذرواژه باید حداقل شش کاراکتر بوده و با تائیدیه گذرواژه یکسان باشد.',
+    'password' => 'گذرواژه باید حداقل هشت کاراکتر بوده و با تائیدیه گذرواژه یکسان باشد.',
```
Comment (andrey-helldar) [inline src/fa/passwords.php]: For Laravel 5.8, use version ^4.0.

**Comment by caouecs** [comment]
thank you

**Comment by andrey-helldar** [comment]
@caouecs, this is mistake. This request makes changes to the master branch, which is used for framework versions up to 5.7 and lower.

**Pull request merged**